tests/fixtures/get-test-server.ts

```diff
@@ -1,12 +1,22 @@
 import { afterEach } from "bun:test"
 import { tmpdir } from "node:os"
 import defaultAxios from "redaxios"
+import type { Options, Response } from "redaxios"
 import { startServer } from "./start-server"
 
+interface CustomAxios {
+  get: <T = any>(url: string, config?: Options) => Promise<Response<T>>
+  post: <T = any>(url: string, data?: any, config?: Options) => Promise<Response<T>>
+  delete: <T = any>(url: string, config?: Options) => Promise<Response<T>>
+  _authToken: string
+  setAuthToken: (token: string) => void
+  clearAuthToken: () => void
+}
```

Comment:
hmm don't use a custom axios instance. It's so simple to change an auth header, no need to complicate what everyone already knows (axios)